### PR TITLE
MGT-1 Management HA test readme

### PR DESCRIPTION
**Pull request by rohit-rp**
https://github.com/openconfig/featureprofiles/issues/2762

### CODE REVIEW

File: feature/system/management/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    *   /network-instances/network-instance[name=default]/protocols/protocol[identifier=BGP, name=BGP]/neighbor[ip=<neigh_ip>]/afi-safis/afi-safi[afi-safi-name=ipv6-unicast]/config/prefix = <lo0 IPv6 Address>
+
+##### Configure linecard ports to ATE using VRRP
+
+*   Configure IPv6 addresses on DUT and ATE ports 3 and 4 from one IPv6 subnet ```vrrp_subnet```
+    *   /interfaces/interface/subinterfaces/subinterface/ipv6/addresses/address/config/ip
+    *   /interfaces/interface/subinterfaces/subinterface/ipv6/addresses/address/config/prefix-length
+*   Configure ATE Port-3 and Port-4 to run VRRP
+    *   Configure a VRRP group-1 with a virtual IP address from the ```vrrp_subnet```, priority 100 and preempt enabled on ATE Port-3
+    *   Configure a VRRP group-1 with the same virtual IP address, priority 120 and preempt enabled on ATE Port-4
```
Comment (LimeHat) [inline feature/system/management/README.md:57]: The usage of VRRP assumes that there's a shared layer-2 segment between the ports. How is it going to be achieved in the context of the test?

Is the ATE device supposed to emulate the shared segment?

File: feature/system/management/README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+### Applying configuration
+
+For each section of configuration below, prepare a gnmi.SetBatch with all the configuration items appended to one SetBatch. Then apply the configuration to the DUT in one gnmi.Set using the `replace` option
+
+### Initial Setup:
+
+*   Connect DUT port-1, 2, 3 and 4 to ATE port-1, 2, 3 and 4
+*   Create VRF "mgmt" on DUT
+    *   /network-instances/network-instance[name=mgmt]/vrfs/vrf[name=mgmt]/config/name = mgmt
+    *   /network-instances/network-instance[name=mgmt]/vrfs/vrf[name=mgmt]/config/route-distinguisher = 64512:100
```
Comment (xw-g) [inline feature/system/management/README.md:22]: Doesn't seems like there is such OC schema https://openconfig.net/projects/models/schemadocs/jstree/openconfig-network-instance.html

cc: @rszarecki

File: feature/system/management/README.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+*   Create an IPv6 networks ```ateNet``` attached to ATE port-1, 2, 3 and 4
+*   Create a loopback interface "lo0" on DUT and assign it an IPv6 address
+    *   /interfaces/interface[name=lo1]/config/name = lo1
+    *   /interfaces/interface[name=lo1]/config/type = softwareLoopback
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/addresses/address/config/ip
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/addresses/address/config/prefix-length
+*   Configure the loopback interface to participate in the VRF "mgmt"
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/vrf/name = mgmt
```
Comment (xw-g) [inline feature/system/management/README.md:30]: I don't think `/ipv6/vrf` OC path is legit? (Same apply to the following OC path that put interface to a VRF)
https://openconfig.net/projects/models/schemadocs/jstree/openconfig-interfaces.html

cc: @rszarecki

File: feature/system/management/README.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol[identifier=BGP, name=BGP]/neighbor[ip=1<neighbor_ip>]/config/peer-as = 64511
+    * /network-instances/network-instance[name=mgmt]/protocols/protocol[identifier=BGP, name=BGP]/neighbor[ip=<neigh_ip>]/afi-safis/afi-safi[afi-safi-name=ipv6-unicast]/config/route-distinguisher = 64512:100 
+*   Set default import and export policy to ```ACCEPT_ROUTE``` for the eBGP sessions
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/default-import-policy
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/default-export-policy
+*   Advertise a default route from ATE to DUT throught both the BGP sessions
+*   Advertise the loopback interface from DUT to ATE through both the BGP sessions
+    *   /network-instances/network-instance[name=default]/protocols/protocol[identifier=BGP, name=BGP]/neighbor[ip=<neigh_ip>]/afi-safis/afi-safi[afi-safi-name=ipv6-unicast]/config/prefix = <lo0 IPv6 Address>
+
+##### Configure linecard ports to ATE using VRRP
```
Comment (xw-g) [inline feature/system/management/README.md:50]: I don't think we need VRRP for linecard port. 

cc: @rszarecki

File: feature/system/management/README.md (hunk 2 above)
Comment (rohit-rp) [reply feature/system/management/README.md:22]: fixed, I made an error copying/pasting paths

File: feature/system/management/README.md (hunk 3 above)
Comment (rohit-rp) [reply feature/system/management/README.md:30]: Fixed

File: feature/system/management/README.md (hunk 4 above)
Comment (rohit-rp) [reply feature/system/management/README.md:50]: ATE does not support VRRP, removing it, I am using a line card port in absence of oob. Just to test the static route operation

**Review by self-maurya — APPROVED** [review]

**Pull request merged**